### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Jaydenmd**
Hey guys, so I didn't do a whole lot to this code, but I was trying to treat it more like an import and using the code given to do the job I have. Hope this helps and makes sense!

### CODE REVIEW

File: source/Development_4.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+          print(image, end="")
+      print()
+
+  def dealCards(self, numCards: int, players: PlayerList, deck: Deck):
+    if numCards * len(players) > deck.size:
+      return False
+    for player in players:
+      for _ in range(numCards):
+        player.addCard(deck.getCard())
+    return True
```
Comment (MidnightBlackHorse17) [inline source/Development_4.py:156]: Hey Jayden! I saw your comment regarding how you are utilizing the code, but I am a little confused as to what you are trying to do.. Can you please direct me to the code you modified?

**Review by MidnightBlackHorse17 — APPROVED** [review]

File: source/Development_4.py (hunk 1 above)
Comment (Jaydenmd) [reply source/Development_4.py:156]: Hey Joanna! So I was trying to learn how to import code from the card games class better. Unfortunately it did not go as well as I'd hoped, but I was trying to take any numbers from my "val" list and add them up if that makes sense

Comment (MidnightBlackHorse17) [reply source/Development_4.py:156]: Thanks for the clarification Jayden!

**Review by ghost — COMMENTED** [review]
Looks good, one thing i noticed was that you had 'Ace' set to 1 in the comment above your card_val list, but have it actually set to 10 in the list itself. Was that intentional?

**Pull request closed without merging**